bin/ghe-backup

```diff
@@ -5,9 +5,10 @@
 #/ the MySQL database, instance settings, GitHub Pages data, etc.
 #/
 #/ OPTIONS:
-#/   -v | --verbose    Enable verbose output.
-#/   -h | --help       Show this message.
-#/        --version    Display version information.
+#/   -v | --verbose       Enable verbose output.
+#/   -h | --help          Show this message.
+#/        --version       Display version information.
+#/        --skip-pages    Skip backup of GitHub Pages artifacts.
```

Comment:
Can we just put `GHE_SKIP_PAGES` in `backup.config`? I think that is the common way we set those variables. (it will be load by `ghe-backup-config`)

Reply:
Done.